README.md

```diff
@@ -1,4 +1,4 @@
-uvcdat
+uv-cdat
```

Comment:
I think @doutriaux1 did it to trigger build.. I will delete it.. thanks for finding it.